main/updater.js

```diff
@@ -0,0 +1,84 @@
+const { BrowserWindow, dialog, ipcMain } = require("electron");
+const { autoUpdater } = require("electron-updater");
+const { showProgressWindow } = require("./progress");
+
+const sendStatusToWindow = text => {
+  const mainWindow = global.windows.mainWindow;
+  mainWindow.webContents.send("messageFromMain", text);
+};
+
+ipcMain.on("cancelUpdate", event => {
+  cancelUpdater();
+});
+
+autoUpdater.allowPrerelease = true;
```

Comment:
It would be better to import the `isDev` module and use `autoUpdater.allowPrerelease = isDev;`.